charts/tap-schema/templates/tap-schema-db-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "tap-schema.fullname" . }}-db
+  labels:
+    app: {{ template "tap-schema.fullname" . }}-db
+{{ include "tap-schema.labels" . | indent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "tap-schema.fullname" . }}-db
+  template:
+    metadata:
+      labels:
+        app: {{ template "tap-schema.fullname" . }}-db
+{{ include "tap-schema.labels" . | indent 8 }}
+    spec:
+      containers:
```

Comment:
Could you add the standard hardening stuff from https://sqr-048.lsst.io/#pod-hardening?